static/local/blackwomenssuffrage/key-figures.md

```diff
@@ -1,83 +1,199 @@
 husband, fellow teacher Robert Heberton Terrell. At the 1890 National Woman Suffrage Assocation Convention, Terrell called attention to the "dual burden" faced by Black women, saying, "A White Woman has only one handicap to overcome&#8212;a great one, true, her sex; a colored woman faces two&#8212;her sex and her race. A colored man has only one&#8212;that of race." In 1892, after a friend of hers was lynched in Memphis by whites, Terrell became involved with anti-lynching activism along with [Ida B. Wells-Barnett](). She helped to found the [National Association of Colored Women](/search?q=National+Association+of+Colored+Women) (NACW) in 1896, and served as its first president until 1901. Terrell believed in the philosophy of "racial uplift," arguing that Black people could advance the cause of racial equality through self-help, including education, hard work and community involvement. She captured this idea in the phrase "Lifting as we climb," which was adopted by NACW as its motto. She led the NACW in creating kindergartens, child care and Mother Clubs to help support working Black women. According to historian Beverly Johnson, she also began to espouse the idea of interracial understanding, believing that education would lead to improved cooperation between races. In 1909, Terrell helped to found the [National Association for the Advancement of Colored People](/search?q=National+Association+for+the+Advancement+of+Colored+People) (NAACP), and the next year, co-founded the [College Alumnae Club](/search?q=College+Alumnae+Club), which later became the National Association of University Women. Terrell believed that women's suffrage was essential to progress for Black women, and worked extensively with both Black and white women's suffrage groups, even picketing the White House with the [National Woman's Party](/search?q=National+Womans+Party). After the passage of the 19th Amendment, she continued to advocate for civil rights and and end to racial discrimination, and her activism continued unabated in her later years. In 1940, she published an autobiography detailing her experience with discrimination, A Colored Woman in a White World. During World War II, she spoke out about the injustice of Black soldiers fighting for freedom abroad while being discriminated against at home. She filed a successful discrimination lawsuit against the American Association of University Women after they denied her entry, and became its first Black member in 1948. She challenged segregation in Washington, D.C. using techniques like boycotts, pickets and sit-ins that would become more popular as the civil rights movement wore on. Perhaps most famously, she and other activists filed a lawsuit&#8212;when she was in her 80s&#8212;challenging segregation in a Washington, D.C., restaurant. This led to a landmark 1953 Supreme Court ruling that outlawed segregation in dining facilities. Terrell died in Maryland in 1954, ending her more than 65-year-long career or activism.
```

Comment:
Look like you forgot to fill in the link here: `[Ida B. Wells-Barnett]()`, I would do` /search?q=Ida+Wells`